Gopkg.toml

```diff
@@ -295,7 +295,8 @@ ignored = [
 
 [[constraint]]
   name = "istio.io/api"
-  branch = "release-1.1"
+  branch = "release-1.1-mcp-enhancements"
```

Comment:
This change will be reverted once https://github.com/istio/api/pull/741 is merged.